hassio/src/components/hassio-card-content.ts

```diff
@@ -17,21 +18,44 @@ class HassioCardContent extends LitElement {
   @property() public hass!: HomeAssistant;
   @property() public title!: string;
   @property() public description?: string;
-  @property({ type: Boolean }) public available?: boolean;
+  @property({ type: Boolean }) public available: boolean = true;
+  @property({ type: Boolean }) public updateAvailable: boolean = false;
```

Comment:
Since this is a generic card (personally not a fan), all properties to this card should be named after how they impact the look of the card and not what they represent in the specific use case that caused them to be added.

Reply:
How about creating a basecard, and and a add-on card that extends it?

Reply:
`<ha-card>` 😆